PoSSapi/src/Domain/Entities/Business.cs

```diff
@@ -0,0 +1,8 @@
+namespace PoSSapi.Domain.Entities;
+
+public class Business
+{
+    public Guid Id { get; set; }
+    public ICollection<BusinessLocation> Location { get; set; }
+    //Needs BusinessType enum but I am unsure what values it could have
```

Comment:
Business type was not mentioned in the QuickFix design document, only in the schemas (and not in any controller :D) (other enum values were mentioned in the doc). I'd quess this could be left out, since it won't be available through the API and the given design does not seem to support services.